docs/developer/migrate/from-ethereum.md

```diff
@@ -63,7 +62,7 @@ event Transfer(address indexed _from, address indexed _to, uint256 _value)
 event Approval(address indexed _owner, address indexed _spender, uint256 _value)
 ```
 
-An ERC20 compliant contract must include the required functions and events at _minimum_.
+An **ERC20 compliant contract** must include the required functions and events at _minimum_.
 It can include additional functions and events and still be ERC20 compliant.
 
 ## The Celo Native Asset and the Celo Dollar
```

Comment:
nit: change Celo Gold to Celo Native Asset in the text

Reply:
@lvpeschke please let me know your feedback on this.

Reply:
Sorry for the long absence - not sure what I understood here initially, but 100% agree with you, fine as it is